app/assets/stylesheets/application.css.scss

```diff
@@ -35,14 +35,32 @@
   height: 100vh;
   position: relative;
 }
+ // h1{
+ //   font-family: 'Poiret One', cursive;
+ // }
 
+.trophy-case-container {
+  background-image: url('http://urlm.co/blog/wp-content/uploads/ipad23.jpg');
+  background-position: center center;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  -webkit-background-size: cover;
+  background-size: cover;
+  height: 100vh;
```

Comment:
Properties should be ordered background-image, background-position, -moz-background-size, -o-background-size, -webkit-background-size, background-size, color, height, position